.github/workflows/deploy.yml

```diff
@@ -13,14 +13,19 @@ jobs:
     name: Cargo Docs
     runs-on: ubuntu-latest
 
+    env:
+      RUSTFLAGS: -Dwarnings
```

Comment:
I don't think this has any effect

Reply:
We use this successfully [in `sentry-rust`](https://github.com/getsentry/sentry-rust/blob/42ff5d1f32b527d773ff12cd43975b58c602cb44/.github/workflows/ci.yml#L52-L53), but I've never used it with cargo doc before. 

CI does not seem to be starting at the moment, so I ran this locally and you're right: For some reason I don't even see the warning when compiling with `RUSTFLAGS=-Dwarnings RUSTUP_TOOLCHAIN=nightly make doc`.

Reply:
Update it does show the warning (it just came up in my scroll buffer), but it doesn't render as error. This is due to [this open issue](https://github.com/rust-lang/rust/issues/43031).

Reply:
Found that it works with `RUSTDOCFLAGS`.

Reply:
Also updated to the cargo action as per suggestion in https://github.com/getsentry/relay/pull/804 since that will report the failures.